tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from expertsystem import ui
+from expertsystem.data import ParticleCollection
+
+
+@pytest.fixture(scope="module")
```

Comment:
Why not `"package"`?

Reply:
Ah nvm, some of the tests modify this instance, right?

Reply:
Yeah exactly